model/qti/asset/AssetManager.php

```diff
@@ -147,6 +151,13 @@ public function importAuxiliaryFiles(QtiResource $qtiItemResource)
         return $this;
     }
 
+    private function isAssetCssFolder(string $relativePath): bool
+    {
+        $relativePathParts = explode('/', dirname($relativePath));
```

Comment:
maybe pathinfo will help you more on your check 
as if you have `./assets/' and you exploded that it will generate a different outcome then the expected one

Reply:
pathinfo will not help much in this case. I'm worried that we are considering only single scenario where our path contain expected structure. I' not sure if this can be simplify this way. What if we have different structure
? Can we be sure that it will always looks similar to this:
```assets/9244e5cf60b621670d905/css/tao-user-styles.css```